packages/vue-searchbox/src/index.js

```diff
@@ -6,19 +6,19 @@ import version from './components/Version/index';
 const components = [SearchBox, SearchBase, SearchComponent];
 
 const install = function(Vue) {
```

Comment:
@anik-ghosh-au7 Why did you change this file?

Reply:
@bietkul reverted the changes